lib/ar2dto/config.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "singleton"
+
+module AR2DTO
+  class Config
+    include Singleton
+
+    def self.reset!
+      instance.except = []
+      instance.class_prefix = ""
+      instance.class_suffix = "DTO"
+    end
+
+    attr_accessor :except, :class_prefix, :class_suffix
+
+    def initialize
+      @except = []
```

Comment:
I wonder if the name of this variable would be better if called something like `excluded_attributes`, `excluded_attrs` or similar.

Reply:
I think Martin used this name to follow the `as_json` API